### PR TITLE
Feat: Added Image components

### PR DESCRIPTION
**Pull request by shuntzhere**
Added 3 types of Images:

- Squared Image
- Rounded Image
- Responsive Image
Screenshot (23)

### CODE REVIEW

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.image-square {
+  width: 100px;
```
Comment (shubhamkumar648) [inline components/image/image.css:2]: suggestion: you can also use relative units % or rem for more responsive

**Review by shubhamkumar648 — COMMENTED** [review]
remaining looks good

File: components/image/image.css (hunk 1 above)
Comment (shuntzhere) [reply components/image/image.css:2]: Thank you for the review, I changed the responsive behaviour for the responsive image but I want to go with fixed sizing for square and round images.

**Review by kjais1720 — APPROVED** [review]
Looks clean, just one suggestion, try to use rems more, instead of px

**Comment by shuntzhere** [comment]
> Looks clean, just one suggestion, try to use rems more, instead of px

Sure, I'll take care of it while refactoring. Thanks

File: components/image/image.css (hunk 1 above)
Comment (shubhamkumar648) [reply components/image/image.css:2]: ok fine

**Review by shubhamkumar648 — APPROVED** [review]

**Pull request merged**